packages/frontend/src/library-mode-entrypoint.tsx

```diff
@@ -36,7 +35,9 @@ export const Root = ({
 }: RootProps) => {
     const config = createConfig({
         chains: [supportedChain] as const,
-        transports: { [supportedChain.id]: supportedChain.transport },
+        transports: {
+            [supportedChain.id]: http(supportedChain.rpcUrls.default.http[0]),
```

Comment:
This is wrong. The RPC URL in this case must be taken directly by the passed in props as we need to reference the local Anvil instance and not the remote chain's RPC.